src/oscar/apps/dashboard/catalogue/views.py

```diff
@@ -201,6 +202,8 @@ def __init__(self, *args, **kwargs):
                          'stockrecord_formset': self.stockrecord_formset}
 
     def dispatch(self, request, *args, **kwargs):
+        if not request.user.is_staff:
+            return HttpResponseForbidden()
```

Comment:
It was required to get these tests passing https://github.com/django-oscar/django-oscar/blob/master/tests/functional/dashboard/catalogue_tests.py#L78. I don't think we can support Django 1.10 unless we drop the tests.